src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -339,13 +339,16 @@ final case class PurposeApiServiceImpl(
       purpose        <- purposeManagementService.getPurpose(purposeUUID)
       tenant         <- tenantManagementService.getTenant(organizationId)
       tenantKind     <- tenant.kind.toFuture(TenantKindNotFound(tenant.id))
-      riskAnalysisForm = purpose.riskAnalysisForm.map(RiskAnalysisConverter.dependencyToApi)
-      _              <- riskAnalysisForm
-        .traverse(RiskAnalysisValidation.validate(_, schemaOnlyValidation = false)(tenantKind))
-        .leftMap(RiskAnalysisValidationFailed(_))
-        .toEither
-        .toFuture
       version        <- getVersion(purpose, versionUUID)
+      riskAnalysisForm = purpose.riskAnalysisForm.map(RiskAnalysisConverter.dependencyToApi)
+      _              <-
+        if (version.state == PurposeManagementDependency.PurposeVersionState.SUSPENDED) Future.unit
+        else
+          riskAnalysisForm
+            .traverse(RiskAnalysisValidation.validate(_, schemaOnlyValidation = false)(tenantKind))
+            .leftMap(RiskAnalysisValidationFailed(_))
+            .toEither
+            .toFuture
```

Comment:
formally we need to validate the risk analysis when the version to activate is in DRAFT.
That will cover not only the `SUSPENDED` state, but also the `WAITING_FOR_APPROVAL` (checking the code, I just noticed that it would cause the same bug).

Plus, we can use cats helper if you prefer (safe because we are working on `Either`)
```suggestion
      _              <- riskAnalysisForm
            .traverse(RiskAnalysisValidation.validate(_, schemaOnlyValidation = false)(tenantKind))
            .leftMap(RiskAnalysisValidationFailed(_))
            .toEither
            .whenA(version.state == PurposeManagementDependency.PurposeVersionState.DRAFT)
            .toFuture
```